### DISCUSSION TITLE
renderToString is not supported in the browser

### DISCUSSION THREAD

**Discussion by almahdi404** [solidjs/solid#1846] [Q&A]
I am actually trying to do ssr, using the [SpiderMonkey](https://firefox-source-docs.mozilla.org/js/index.html) JavaScript engine.

but **solid-js** says `renderToString is not supported in the browser`

Is there any way around for this?

**Comment by atk** [comment]
You need to import solid with the node condition.

**Comment by jrood** [comment]
@atk how does one import with the node condition?

**Reply by atk** [reply]
In node, you would use `--conditions server`, but I don't know the specifics of your runtime. If it doesn't support module conditions, you might need to use aliased imports.

**Reply by jrood** [reply]
thanks!
